config/identity/config.yaml

```diff
@@ -12,11 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-define:
-  - &github-type "github-workflow"
-  - &gitlab-type "gitlab-pipeline"
-  - &codefresh-type "codefresh-workflow"
-  - &buildkite-type "buildkite-job"
```

Comment:
Note that some tests were failing due to parse errors related to these anchors. If we merge this, we should also check public-good-instance/scaffolding for usage of yaml anchors.